lib/upland_mobile_commons_rest/profiles.rb

```diff
@@ -1,8 +1,19 @@
 module UplandMobileCommonsRest
   class Profiles < Base
+    def base_path
+      'profile'
+    end
+
+    def get(phone_number)
+      resp = client.get_request("#{base_path}?phone_number=#{phone_number}")
```

Comment:
Need to cgi escape the phone_number before using it in a url for security against server side request forgery, since phone number is likely from user input.